x-pack/docs/en/security/troubleshooting.asciidoc

```diff
@@ -319,6 +320,65 @@ In this case, you must install the
 <<ciphers, JCE Unlimited Strength Jurisdiction Policy Files>>.
 --
 
+[[trb-security-kerberos]]
+=== Common Kerberos exceptions
+
+*Symptoms:*
+
+* User authentication fails due to either GSS negotiation failure
+or a service login failure on the server side or in the {es} HTTP client. Some of
+the common exceptions are listed below with some tips to resolve them. 
```

Comment:
If I'm understanding the first sentence correctly, I think it should be changed to something like this: "User authentication fails due to either a GSS negotiation failure or a service login failure (on the server or the {es} HTTP client)".